### DISCUSSION TITLE
Group album titles regardless of artists

### DISCUSSION THREAD

**Discussion by Batwam** [navidrome/navidrome#2899] [Ideas]
Hello, I noticed that Navidrome splits songs from the same album into separate albums if their artists are different. Apparently [this is a feature](https://www.navidrome.org/docs/faq/#:~:text=Navidrome%20only%20organises%20music%20by,be%20set%2C%20for%20all%20tracks.) and I should retag my library to suit. I haven't seem other music apps doing this. For instance, Rhythmbox or even Plex group the songs from the same album together.

I'm guessing that this is done to avoid grouping songs from different artists who somehow decided to name their album the same way? On the basis that this risk is remote (non-existent in my case), would it be possible to have at least and option to group song with the same album together, regardless of the artist(s)?

**Comment by metalheim** [comment]
I mean that's what the albumartist Tag is for.

With what youre suggesting, navidrome would group all Songs together that have the same Album name. And Album names are faaaar from unique. Theres thousands of albums that have the same name.
So imho, navidromes current behaviour is the correct one and handles all cases well.

**Reply by Batwam** [reply]
Thanks for the quick response. I didn't know this solution but I think that this is the correct way of doing this rather than mark as Compilation as suggested in the doc, at least when there is a main artist and various other collaborators of each song.

I populated `albumartist` to be the main artist, now all songs are grouped without having to mark as compilation so, that's great, thanks!

**Reply by deluan** [reply]
Glad your issue was resolved. If you think the docs are misleading, incomplete or otherwise wrong, a contribution would be great ;)
